src/app/components/Banner.tsx

```diff
@@ -0,0 +1,92 @@
+import { Body, Box, Flex, Image } from '@mycrypto/ui';
+import type { PropsWithChildren } from 'react';
+import { useState } from 'react';
+
+import info from '@assets/icons/alert-grey.svg';
+import error from '@assets/icons/alert-red.svg';
+import caret from '@assets/icons/caret.svg';
+import success from '@assets/icons/circle-checkmark.svg';
+import warning from '@assets/icons/circle-warning.svg';
+import action from '@assets/icons/queue-waiting.svg';
+import { theme } from '@theme';
+
+type BannerType = keyof typeof theme.variants.banner;
+
+interface BannerProps {
+  type: BannerType;
+  label: string;
+  banner?: string;
+  extended?: boolean;
+}
+
+const icons: { [key in BannerType]: string } = {
+  success,
+  info,
+  action,
+  warning,
+  error
+};
+
+export const InnerBanner = ({ type, children }: PropsWithChildren<Pick<BannerProps, 'type'>>) => (
+  <Box backgroundColor={theme.variants.banner[type].color} sx={{ borderRadius: 'banner' }}>
```

Comment:
We should use the theme from the provider here. Otherwise it will make it difficult to change down the road